src/api/v1/endpoints/skill.py

```
@@ -0,0 +1,52 @@
+from fastapi import APIRouter
+
+from src.models.response_schema import SuccessResponse
+from src.models.skill_schema import Drill, SkillLabRequest, SkillLabResponse
+
+router = APIRouter()
+
+
+@router.post("/", response_model=SuccessResponse[SkillLabResponse])
+async def create_skill_routine(
+    request: SkillLabRequest,
+) -> SuccessResponse[SkillLabLResponse]:
```

Ruff (0.15.0)

[error] Undefined name `SkillLabLResponse`

(F821)